dockerfiles/theia/instructions/runtime-configure.run

```diff
@@ -0,0 +1,30 @@
+    echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers \
+    # Create /projects for Che
+    && mkdir /projects \
+    # Create root node_modules in order to not use node_modules in each project folder
+    && mkdir /node_modules \
+    # Download yeoman generator plug-in
+    && curl -L -o /default-theia-plugins/theia_yeoman_plugin.theia https://github.com/eclipse/theia-yeoman-plugin/releases/download/untagged-c11870b25a17d20bb7a7/theia_yeoman_plugin.theia \
+    # Download vscode git plug-in
+    && curl -L -o /default-theia-plugins/vscode-git-1.3.0.1.vsix https://github.com/che-incubator/vscode-git/releases/download/1.30.1/vscode-git-1.3.0.1.vsix \
+    && for f in "${HOME}" "/etc/passwd" "/etc/group /node_modules /default-theia-plugins /projects"; do\
+           sudo chgrp -R 0 ${f} && \
+           sudo chmod -R g+rwX ${f}; \
+       done \
+    && cat /etc/passwd | sed s#root:x.*#root:x:\${USER_ID}:\${GROUP_ID}::\${HOME}:/bin/bash#g > ${HOME}/passwd.template \
+    && cat /etc/group | sed s#root:x:0:#root:x:0:0,\${USER_ID}:#g > ${HOME}/group.template \
+    # Add yeoman, theia plugin generator and typescript (to have tsc/typescript working)
+    && yarn global add yo @theia/generator-plugin@0.0.1-1562578105 typescript@2.9.2 \
+    && mkdir -p ${HOME}/.config/insight-nodejs/ \
+    && chmod -R 777 ${HOME}/.config/ \
+    # Disable the statistics for yeoman
+    && echo '{"optOut": true}' > $HOME/.config/insight-nodejs/insight-yo.json \
+    # Link yarn global modules for yeoman
+    && local_modules=$(ls -d1 /usr/*/node_modules 2>/dev/null || ls -d1 /usr/*/*/node_modules) \
+    && mv ${local_modules}/* /usr/local/share/.config/yarn/global/node_modules && rm -rf ${local_modules} && ln -s /usr/local/share/.config/yarn/global/node_modules $(dirname ${local_modules})/ \
+    # Cleanup tmp folder
+    && rm -rf /tmp/* \
+    # Cleanup yarn cache
+    && yarn cache clean \
+    # Change permissions to allow editing of files for openshift user
+    && find ${HOME} -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \;
```

Comment:
I think we already did this step on line 10. Why do it twice? Or why do it earlier if you have to do it again? Maybe we omit ${HOME} on line 10?